src/v2/guide/class-and-style.md

```diff
@@ -91,69 +91,69 @@ data: {
 }
 ```
 
-Which will render:
+Lo cual va mostrar:
 
 ``` html
 <div class="active text-danger"></div>
 ```
 
-If you would like to also toggle a class in the list conditionally, you can do it with a ternary expression:
+Si también quieres puedes cambiar una lista de clases condicionalmente, lo puedes hacer con una expresión ternaria:
 
 ``` html
 <div v-bind:class="[isActive ? activeClass : '', errorClass]"></div>
 ```
 
-This will always apply `errorClass`, but will only apply `activeClass` when `isActive` is `true`.
+Esto siempre se aplicará`errorClass`, pero solo aplicará `activeClass` cuando `isActive` es `true`.
 
-However, this can be a bit verbose if you have multiple conditional classes. That's why it's also possible to use the object syntax inside array syntax:
+Sin embargo, esto puede ser muy detallado si varias clases condicionadas. Esto es porque también es posible usar la sintaxis de un objeto dentro de la sintaxis de un arreglo:
 
 ``` html
 <div v-bind:class="[{ active: isActive }, errorClass]"></div>
 ```
 
-### With Components
+### Con componentes
 
-> This section assumes knowledge of [Vue Components](components.html). Feel free to skip it and come back later.
+> Esta sección asume que tienes conocimiento de [Componentes Vue](components.html). Siéntete libre de saltártela y volver después.
 
-When you use the `class` attribute on a custom component, those classes will be added to the component's root element. Existing classes on this element will not be overwritten.
+Cuando usas el atributo `class` en un componente personalizado, estas clases serán agregadas a el elemento raíz del componente. Clases existentes en este elemento no serán sobreescritas.
 
-For example, if you declare this component:
+Por ejemplo, si tu declaras este componente:
```

Comment:
Sounds natural: `Por ejemplo, si declaras este componente`… just remove the `tu`.